src/alda/sound.clj

```diff
@@ -121,38 +134,66 @@
 
 (def ^:dynamic *play-opts* {})
 
-; TODO: control where to start and stop playing using the start & end keys
+(defmacro with-play-opts
+  "Update *play-opts* w"
```

Comment:
It looks like you may have been kidnapped by [Candlejack](http://knowyourmeme.com/memes/candlejack) while you were wri